lib/big.js

```diff
@@ -85,6 +85,11 @@ window.addEventListener('load', function() {
   slideDivs = slideDivs.map(function(slide) {
     slide.setAttribute('tabindex', 0);
     slide.classList.add('slide');
+
+    if(slide.hasAttribute('data-background-image')){
+      preloadImage(slide.getAttribute('data-background-image'));
```

Comment:
inline this? Not seeing the need for a separate function

Reply:
Also perhaps it'd be better to use [preload links](https://www.w3.org/TR/preload/#introduction) rather than creating Img objects, to better prioritize the requests.

Reply:
I removed the separate function.
Also, great idea with the preload links. have changed to using this instead.
Since the if-block became bigger its suitable for a separate function again?